testing/MLDB-1359_procedure_latest_run.py

```diff
@@ -0,0 +1,59 @@
+#
+# MLDB-1359_procedure_latest_run.py
+# Mich, 2016-02-05
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+#
+import time
+from dateutil import parser as date_parser
+
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+
+class ProcedureLatetRunTest(MldbUnitTest): # noqa
```

Comment:
Ah right.  The validation is done before calling the latestrun endpoint.  This is why the code does not segfault in the static cast.  No need to add a test then.